Formula/bombadillo.rb

```diff
@@ -0,0 +1,32 @@
+class Bombadillo < Formula
+  desc "Non-web browser, designed for a growing list of protocols"
+  homepage "http://bombadillo.colorfield.space"
+  url "https://tildegit.org/sloum/bombadillo/archive/2.3.0.tar.gz"
+  sha256 "0b98dfdfcb6499b238f5878f4780f595b25bc4b6e7afb76eabbd907c6d74d71f"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    require "pty"
+
+    begin
+      cmd = bin/"bombadillo gopher://bombadillo.colorfield.space"
+      config = testpath/".config"
+      r, w, pid = PTY.spawn("stty rows 80 cols 43 && XDG_CONFIG_HOME=#{config} #{cmd}")
+      sleep 1
+      w.write "q"
+      assert_match /Bombadillo is a non-web browser/, r.read
+
+      status = PTY.check(pid)
+      assert_not_nil status
+      assert_true status.success?
+    ensure
+      # Ensure we don't leave the process around, just in case
+      Process.kill("TERM", pid) if status.nil?
+    end
```

Comment:
This isn't needed, brew will do this

Reply:
Thanks -- I took out the `Process.kill`.